src/driver/franka_plan_runner.cc

```diff
@@ -863,10 +894,8 @@ franka::JointPositions FrankaPlanRunner::JointPositionCallback(
       return franka::MotionFinished(output_to_franka);
     }
 
-    {
-      // make a copy and pass by value
-      const auto plan_utime {plan_utime_};
-      auto overtime_warning {[plan_utime]() {
+    {  // threaded logging, capture member vars by val
+      auto overtime_warning {[plan_utime = plan_utime_]() {
```

Comment:
Where is this actually threaded?

Reply:
Oh nvm didn't see if with collapsed diff

Reply:
yep L836 `std::thread overtime_warning_thread {overtime_warning};`